src/database/entities/cart.entity.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  Column,
+  Entity,
+  PrimaryGeneratedColumn,
+  OneToMany,
+  JoinColumn,
+} from 'typeorm';
+import { CartItem } from './cart-item.entity';
+
+export enum CartStatus {
+  OPEN = 'OPEN',
+  ORDERED = 'ORDERED',
+}
+
+@Entity({ name: 'carts' })
+export class Cart {
+  @PrimaryGeneratedColumn('uuid')
+  id?: string;
+
+  @Column({ type: 'date', nullable: false })
+  user_id: string;
+
+  @Column({ type: 'timestamp' })
+  created_at?: number;
+
+  @Column({ type: 'timestamp', nullable: false })
+  updated_at: number;
+
+  @Column({
+    type: 'enum',
+    enum: [CartStatus.OPEN, CartStatus.ORDERED],
```

Comment:
Not so clear why you need an array of values instead of use your CartStatus enum here.